packages/config/config/devices/0x0086/templates/aeotec_template.json

```
@@ -0,0 +1,1132 @@
+		"label": "Automatic Report: Group 3 - kWh (Clamp 2)",
+		"valueSize": 4
+	},
+	"auto_report_group3_kwh_clamp3": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Automatic Report: Group 3 - kWh (Clamp 3)",
+		"valueSize": 4
+	},
+	"auto_report_group3_kwh_wholehem": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Automatic Report: Group 3 - kWh (Whole HEM)",
+		"valueSize": 4
+	},
+	"device_tag": {
+		"label": "Device Tag",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"partner_id_aeotec": {
+		"label": "Partner ID",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 0,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Aeotec",
+				"value": 0
+			}
+		]
+	},
+	"partner_id_att": {
+		"label": "Partner ID",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Aeotec",
+				"value": 0
+			},
+			{
+				"label": "AT&T",
+				"value": 1
+			}
+		]
+	},
+	"partner_id_other": {
+		"label": "Partner ID",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Aeotech",
+				"value": 0
+			},
+			{
+				"label": "Other",
+				"value": 1
+			}
+		]
+	},
+	"wattage_trigger": {
+		"label": "Minimum change in wattage to trigger event",
+		"description": "Minimum change in wattage to trigger event",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 32000,
+		"defaultValue": 50,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"wattage_percent_trigger": {
+		"label": "Minimum change in wattage to trigger event",
+		"valueSize": 1,
+		"unit": "%",
+		"minValue": 0,
+		"maxValue": 100,
+		"defaultValue": 10,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"external_switch_two": {
+		"$import": "../../templates/master_template.json#base_0-1_nounit",
+		"label": "External Switch Type",
+		"options": [
+			{
+				"label": "Momentary mode",
+				"value": 0
+			},
+			{
+				"label": "Switch mode",
+				"value": 1
+			}
+		]
+	},
+	"external_switch_unidentified": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "External Switch Type",
+		"options": [
+			{
+				"label": "Momentary mode",
+				"value": 0
+			},
+			{
+				"label": "Switch mode",
+				"value": 1
+			},
+			{
+				"label": "Unidentified mode",
+				"value": 255
+			}
+		]
+	},
+	"external_switch_3": {
```

```suggestion
	"external_switch_3way": {
```
would be a tad clearer when you're looking at the files that use this.